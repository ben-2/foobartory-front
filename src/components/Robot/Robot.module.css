.robot {
  width: 100px;
  min-height: 100px;
  margin-right: 10px;
  margin-bottom: 20px;
  display: inline-block;
}