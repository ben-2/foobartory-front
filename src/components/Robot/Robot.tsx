import styles from './Robot.module.css';
import { Card } from '@material-ui/core';

export function Robot() {
  return (
    <>
      <Card className={styles.robot}>
        Robot
      </Card>
    </>
  );
}
