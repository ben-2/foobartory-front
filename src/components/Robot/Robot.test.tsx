import { Robot } from './Robot';
import { shallow } from 'enzyme';

describe('<Robot />', () => {
  it('renders an emtpy Paper Card when the game starts', () => {
    const wrapper = shallow(<Robot />)
    expect(wrapper).toMatchSnapshot();
  })
});
