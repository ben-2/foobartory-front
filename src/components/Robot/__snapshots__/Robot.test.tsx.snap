// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Robot /> renders an emtpy Paper Card when the game starts 1`] = `ShallowWrapper {}`;
